docs/developer.rst

```diff
@@ -144,6 +144,22 @@ Pull requests submitted to the repository will automatically be tested using
 these systems and results reported in the ``checks`` section of the pull request
 page.
 
+Updating conda-forge retriever-feedstock
+========================================
+
```

Comment:
Just to be safe, we should probably preface this paragraph that we took from conda-forge/retriever-feedstock by saying something like: 
```
To update the confa-forge retriever-feedstock, please reference the updating retriever-feedstock 
section from the 'conda-forge/retriever-feedstock <https://github.com/conda-forge/retriever-feedstock>'_ repository: 
"
```
and start the paragraph with quotes.